tool/vfx/vfxSection.h

```diff
@@ -1081,6 +1081,7 @@ class SectionColorBuffer : public Section
         Section(m_addrTable, MemberCount, SectionTypeUnset, "colorBuffer")
     {
         memset(&m_state, 0, sizeof(m_state));
+        m_state.channelWriteMask = 0xf;
```

Comment:
0xf -> 0xF, we use uppercase for hex numbers.

Reply:
Sure, will update this when submit this change.